autoprecompiles/src/lib.rs

```diff
@@ -335,7 +336,10 @@ const PC_LOOKUP_BUS_ID: u64 = 2;
 const RANGE_CHECK_BUS_ID: u64 = 3;
 
 impl<T: FieldElement> Autoprecompiles<T> {
-    pub fn build(&self) -> (SymbolicMachine<T>, Vec<BTreeMap<String, String>>) {
+    pub fn build(
+        &self,
+        bus_interaction_handler: impl BusInteractionHandler<T> + 'static,
```

Comment:
does this have something default that can be easily built if the user of this function is not interested? Or could be optional maybe?

Reply:
ah I see in the other PR that it does have default